pkg/stanza/docs/types/cache.md

```diff
@@ -0,0 +1,34 @@
+# Parsing Cache
```

Comment:
I can appreciate the detailed documentation. However I think we need to keep in mind that the intended reader should be _users_ of the collector, who are quite often not developers of the collector. For that reason, I think the documentation should describe only functionality that a user may observe and does not need to give detail about the code-level implementation.